docs/configuration/source-config.md

```diff
@@ -64,11 +64,14 @@ The Kafka source consumes a `topic` using the client library [librdkafka](https:
 - `bootstrap.servers`
 Comma-separated list of host and port pairs that are the addresses of a subset of the Kafka brokers in the Kafka cluster.
 
+- `auto.offset.reset`
```

Comment:
@tuziben new parameter for the Kafka source.

Reply:
This is not a Quickwit-specific parameter but a Kafka one that you can pass to the client, via the `client_params` property. For instance:

```yaml
version: 0.6
source_id: my-kafka-source
source_type: kafka
params:
  topic: my-topic
  client_params:
    auto.offset.reset: earliest
    bootstrap.servers: localhost:9092
    security.protocol: SSL
```